tabbycat/results/result.py

```diff
@@ -106,6 +106,9 @@ def DebateResult(ballotsub, *args, **kwargs):  # noqa: N802 (factory function)
     r = kwargs.pop('round', ballotsub.debate.round)
     tournament = kwargs.pop('tournament', r.tournament)
     result_class = get_result_class(ballotsub, r, tournament)
+    if result_class.uses_speakers and 'criteria' not in kwargs:
+        from .models import ScoreCriterion
+        kwargs['criteria'] = ScoreCriterion.objects.filter(tournament=tournament)
```

Comment:
Would 
```suggestion
        kwargs['criteria'] = tournament.scorecriterion_objects.all()
```
work?

(I'm guessing the keyword argument is just to allow a performance optimization, and would never be anything other than all the criteria?)